src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,84 @@
+package duke;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+/**
+ * TaskList is a class that contains the task list.
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    /**
+     * Constructor for duke.TaskList, which stores task entered by user in duke.Duke.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
```

Comment:
you could consider removing the `this` modifier since the variable `tasks` is not being shadowed (as advised by the course coding standard). This is present in a few places in the codebase too.